templates/blog/post_detail.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <h2>{{ post.title }}</h2>
```

Comment:
The <h2> tag should not be inside the <head> section. It should be placed within the <body> section to ensure proper HTML structure.